security/tools/generate_cert/main.go

```diff
@@ -55,6 +55,7 @@ var (
 	outPriv        = flag.String("out-priv", "priv.pem", "Output private key file.")
 	keySize        = flag.Int("key-size", 2048, "Size of the generated private key")
 	mode           = flag.String("mode", selfSignedMode, "Supported mode: self-signed, signer, citadel")
+	isServer       = flag.Bool("server", false, "Whether this certificate is for a server.")
```

Comment:
To help users understand and use this flag, can you add a comment on the difference between a server certificate and a non-server certificate?

Reply:
added a comment to explain a little bit